### ISSUE TITLE
Don't recognize similiar monitors setup's with different resolutions

### ISSUE THREAD

**Issue by Bruno1978** [terrymacdonald/DisplayMagician#34]
Hi.
I have 4 monitor's in my setup, 3x 27" QHD and 1 22" FHD on top. 
The issue is i have um profile without surround on QHD resolution, and other profile with surround with the 3x 27" monitors on FHD resolution, the issue has i want to have a another profile without surround for FHD resolution and the program don't recognized the setup whitout surround in FHD resolution it's show has the setup with QHD resolution.

**Comment by terrymacdonald** [comment]
Hi @Bruno1978,

Not sure if you've read some of the other issues here, but the current version of DisplayMagician (v1.1.4) doesn't detect or store all of the display settings. So to fix that problem I've been developing some new display manipulation libraries that will detect and store all the settings.

These are currently still being integrated into DisplayMagician v2.0.0, and will likely be realised in a month or so.

As a work around, I suggest you look at NVIDIAInfo. I created NVIDIAInfo just to allow me to test the new display manipulation libraries, but it might help you with your problem until DisplayMagician v2.0.0 is released.

You can download NVIDIAInfo from https://github.com/terrymacdonald/NVIDIAInfo/releases/tag/v1.0.0

Hope that helps
Terry

**Comment by terrymacdonald** [comment]
And another update. This has been a monster challenge, but I think I'm nearing the end of the NVIDIA chapter at least. Things that the new DisplayMagician can do:

Swap between surround and non-surround profiles within about 14 seconds
Handle surround and non-surround screens together
Works with NVIDIA mosaic cards now
Uses the PCI video card vendor ID to recognise NVIDIA and AMD vendors, and sets it's driver mode accordingly
Can be 'forced' to use a different video card mode if needed (unlikely to be used, but hey, why not)
Can set/clear a Desktop Background per profile if you want
Stores and applies Windows HDR and NVIDIA HDR settings
Records all settings provided by the NVIDIA driver and windows drivers under Windows 10, meaning that you display refresh settings are stored and set, as are bezel distances, or any other weird and wonderful settings you creative users are able to configure :)
But at present there are still bits I have to finish before I can craft any sort of production ready release:

It is very slow to load. I've identified this is due to DisplayMagician keeping on calling the video card libraries. I need to implement some caching to ensure that the video detection is only run at programme launch and when screen layout changes in some way.
I need to add AMD library support. This is the core reason why I started this journey, so it seems a little disingenuous to release DisplayMagician without this!
I need to add a nice message that popups when an old DisplayMagician file is detected, that informs the user that this is a new version of DisplayMagician that uses a brand new DisplayProfile format that will require them to create a brand new set of Display Profile layouts.
I need to test, and to provide my beta testers an early release of the software so that I iron out any bugs before a general release.
I need to add the ability to test the upgrade mechanism without upgrading the production upgrade functionality. This is needed so I can iron out any issues before a general release. Very important as DisplayMagician now has over 1500 users!
And I think that's about it!

Some cool things to come for sure, and it's taken a lot of hard work to get this close. Another little while and we're hopefully good to go!

Thanks
Terry

**Comment by terrymacdonald** [comment]
Great news! DisplayMagician v2.0.1 has been released, and fixes this very issue. Please test it out, and if you still have any issues please log a new issue.

I'm closing this issue as the fix is in DisplayMagician v2.0.1, and I need to clear out the completed issues so I am able to work on the new issues that come in.

Thanks
Terry
